tools/kore-proof-trace/main.cpp

```diff
@@ -42,8 +42,9 @@ int main(int argc, char **argv) {
 
   if (use_streaming_parser) {
     std::ifstream file(input_filename, std::ios_base::binary);
-    proof_trace_file_buffer buffer(file);
-    llvm_rewrite_trace_iterator it(buffer, header);
+    auto *buffer = new proof_trace_file_buffer(std::move(file));
+    llvm_rewrite_trace_iterator it(
+        std::unique_ptr<proof_trace_file_buffer>(buffer), header);
```

Comment:
nitpick: same as above

```suggestion
    auto buffer = std::make_unique<proof_trace_file_buffer>(std::move(file));
    llvm_rewrite_trace_iterator it(buffer, header);
```